src/app/components/feed/FeedTopBar.js

```diff
@@ -25,7 +24,8 @@ const FeedTopBar = ({
   const hasList = Boolean(feed.saved_search);
 
   const handleClick = () => {
-    browserHistory.push(`/${team.slug}/list/${feed.saved_search.dbid}`);
+    const customListDbid = feed.current_feed_team.saved_search ? feed.current_feed_team.saved_search?.dbid : feed.saved_search.dbid;
```

Comment:
suggestion: this can be made simpler:

`const customListDbid = feed.current_feed_team.saved_search?.dbid || feed.saved_search.dbid;`